database/migrations/2022_11_26_183251_increment_user_i_ds.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use App\Models\BaseAlbumImpl;
+use App\Models\Photo;
+use App\Models\User;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class IncrementUserIDs extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up(): void
+	{
+		/** @var User $user */
+		$user = User::find(0);
```

Comment:
I am not sure if we should use `User::` or `DB::table`
The later would make us agnostic from the production code which is likely to be better.